src/conversions/hashbrown.rs

```diff
@@ -51,6 +54,29 @@ where
     }
 }
 
+impl<'py, K, V, H> IntoPyObject<'py> for hashbrown::HashMap<K, V, H>
```

Comment:
Yes, I think it would be desirable to provide these, since there is no real reason why one should not be able to continue using such a container after the conversion to Python.

I vaguely remember that I tried to add them at some point but ran into lots of HRTBs, I just tried again and it worked like I would have expected so either the implementation evolved and made it simpler, or I just did it wrong last time 🤷. 

Since this is already quite a lot I suggest to land them separately.